app/controllers/admin/categories_controller.rb

```diff
@@ -25,8 +25,7 @@ def destroy
 
   def new_or_edit
     @categories = Category.find(:all)
-    @category = Category.new
-    @category = Category.find(params[:id]) if params[:id]
+    params[:id] ? @category = Category.find(params[:id]) : @category = Category.new
```

Comment:
Yep! I think you can simplify it even more by putting the `@category = ` at the start of the line